aws-https/lib/aws-https.ts

```diff
@@ -110,13 +107,22 @@ export class AwsHttps {
                         resolve(null);
                     }
                 });
+
+                // Theoretically this should be called based on requestOptions#timeout.
+                // It doesn't, but leaving this code here in case in ever gets fixed by Node.js.
```

Comment:
Possible typo. (I feel like a human clippie typing this right now.)

                // It doesn't, but leaving this code here in case in ever gets fixed by Node.js.

Did you mean?

                // It doesn't, but leaving this code here in case it never gets fixed by Node.js.

Reply:
The original comment is correct.